hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsAsyncQueueService.kt

```diff
@@ -55,7 +55,7 @@ open class HmppsAsyncQueueService(
 
     val messages = (1..messageCount)
       .asFlow()
-      .map { sqsAsyncDlqClient.receiveMessage(ReceiveMessageRequest.builder().queueUrl(dlqUrl).maxNumberOfMessages(1).attributeNames(QueueAttributeName.ALL).build()) }
+      .map { sqsAsyncDlqClient.receiveMessage(ReceiveMessageRequest.builder().queueUrl(dlqUrl).maxNumberOfMessages(1).messageAttributeNames("All").build()) }
```

Comment:
not sure what was going on here - but one of the tests kept failing as there were no message attributes appearing.  Changed to match the `HmppsQueueService`

Reply:
Strange.